gemviz/tests/test_aboutdialog.py

```diff
@@ -0,0 +1,20 @@
+from PyQt5 import QtWidgets
+
+from .. import aboutdialog
+
+
+def test_about_starts(qtbot):
```

Comment:
Let's add a `requirements-dev.txt` file to describe packages needed to develop this code.  A separate file will be used to describe requirements to build the documentation.

Reply:
Partly, content is documented in the (new) `testing.md` file.

Reply:
Converted `.md` to .rst` and added to docs